contracts/strategies/LPOptimiserStrategy.sol

```diff
@@ -25,12 +27,30 @@ contract LPOptimiserStrategy is DefaultAccessControl, ILpCallback {
     IVoltzVault internal _vault;
     uint256[] internal _pullExistentials;
     IMarginEngine internal _marginEngine;
+    IPeriphery internal _periphery;
+    IVAMM internal _vamm;
     IVoltzVault.TickRange internal _currentPosition;
 
     // MUTABLE PARAMS
 
     uint256 _lastSignal;
     uint256 _lastLeverage;
+    uint256 _k_unwind_parameter; // parameter for k*leverage (for unwinding so this needs to be sent to the contract vault but not used in the strategy vault)
```

Comment:
_lastSignal, _lastLeverage -- not used at all. These were used for trading strategy.